### ISSUE TITLE
All backgrounds of the theme being very light makes it very hard to find out where also light delimiters are

### ISSUE THREAD

**Issue by kupiqu** [PapirusDevelopmentTeam/adapta-kde#25]
All backgrounds of the theme are basically white and delimiters are also basically invisible. This makes it very hard to find hover regions.

Please see the screenshot for an example of this (you can infer where the delimiter is by finding the special mouse icon, which is on the left-down quadrant):

![image](https://user-images.githubusercontent.com/3875196/32850595-962cf84a-ca00-11e7-83b7-53861dc14822.png)

In Breeze some backgrounds are gray, which makes the hover region more salient.

**Comment by varlesh** [comment]
And? But you see this! if you don't like, you can change kate colorscheme or fix and add stroke for all frames

**Comment by varlesh** [comment]
@kupiqu try ask about that on Kvantum dev @tsujan.
See, other apps use stroke for window frame:
![image](https://user-images.githubusercontent.com/8083855/32854078-ea6e9d5e-ca56-11e7-9fc5-23c3176f4bec.png)

**Comment by varlesh** [comment]
@tsujan why Kate not use stroke for window widget, it's bug?

**Comment by kupiqu** [comment]
EDIT: I just read your new comment. So this could be a bug in Kate? I opened a bug in Kate (https://bugs.kde.org/show_bug.cgi?id=386953), although it was for adding the option to change the default background color in plugins. I could add this discussion there too so we figure out what is going on to this respect...

I really like Adapta-kde, otherwise I wouldn't use it, so I prefer not to use a different colorscheme in general :)

Neither in Kate, because a different colorscheme in Kate would make it look "anormal" in my system.

On the other hand, I think that functionality is very important.

Ideally, perhaps the best would be to make the separator line more salient (would that be possible somehow?). 

Changing the background color is an alternative, but it would be more drastic.

Just trying to help here, to achieve a more functional colorscheme, if possible :)

**Comment by tsujan** [comment]
> @tsujan why Kate not use stroke for window widget, it's bug?

If by "stroke" you mean palette, that's intentional -- FeatherPad doesn't follow the Qt style for its view either -- and it is NOT a bug.

Kate's config:

![kate](https://user-images.githubusercontent.com/981076/32854780-fef9b09e-ca54-11e7-8175-c762b21d29a7.png)


FeatherPad's:

![featherpad](https://user-images.githubusercontent.com/981076/32854789-03e7652e-ca55-11e7-8f25-496d503305b0.png)

**Comment by kupiqu** [comment]
I think the colors from Kate config do not apply to Kate plugins, do they?

**Comment by tsujan** [comment]
I don't know about Kate's plugin but it's good for a text-editor to define its view color(s) -- and not any other color -- irrespective to the current style.

**Comment by kupiqu** [comment]
Sure, I agree, but that's not really the issue, is it? I mean, the problem is the transition between the miniview scrollbar and the plugin background. Changing the colors of the text editor have no impact on making that transition more salient (unless the background color also affects the scrollbar, which does not seem obvious).

I have actually double-checked for this, and yes, the background of the miniview scrollbar follows the background of the text-editor, so that's a way to go, although having to change the white background of the text-editor for this doesn't seem the best option...

**Comment by varlesh** [comment]
@tsujan 
> If by "stroke" you mean palette, that's intentional -- FeatherPad doesn't follow the Qt style for its view either -- and it is NOT a bug.

No, see on my screen - Kate not have stroke on window (compare with muon):
![image](https://user-images.githubusercontent.com/8083855/32855767-2917c648-ca5c-11e7-8c36-f5fe9381aa10.png)

**Comment by varlesh** [comment]
![image](https://user-images.githubusercontent.com/8083855/32855852-7407d53a-ca5c-11e7-8c17-acb616769224.png)

**Comment by tsujan** [comment]
So, by "stroke" you mean "borders". First, that doesn't depend on any style engine but on the color scheme. Second, Kate does have border where it's needed. In the following screenshot you see borders where they're needed, i.e., around the side-pane. However, the right area is not a re-sizable pane, so it doesn't need a border. That's how Kate is designed and I think it's OK. Anyhow, it doesn't have anything to do with Kvantum.

![kate1](https://user-images.githubusercontent.com/981076/32856266-8f6f42ca-ca59-11e7-97c2-db11a455a9df.png)

**Comment by varlesh** [comment]
@tsujan Sorry for that... you right!
@kupiqu Breeze not use stroke too. Why you not check this?
![image](https://user-images.githubusercontent.com/8083855/32856310-df92a6f8-ca5d-11e7-83df-088001fa5016.png)

**Comment by varlesh** [comment]
Kate not use border/stroke on scroll map...

**Comment by varlesh** [comment]
@kupiqu i not understan what you mean  "anormal". All looks fine:
![image](https://user-images.githubusercontent.com/8083855/32856619-bb356d76-ca5e-11e7-85e9-75b6aea94e29.png)

**Comment by kupiqu** [comment]
Please remember that the issue is on the transition between the scrollbar and Kate plugins. No one of your screenshots are showing any Kate plugin, please see my screenshot to see this...

In Breeze this is not an (important) issue because the background is darker so the transition between the scrollbar and Kate plugins is much more salient.

![image](https://user-images.githubusercontent.com/3875196/32856830-f80d9110-ca13-11e7-9539-86c97c0900df.png)

**Comment by kupiqu** [comment]
Compare the previous to this one:

![image](https://user-images.githubusercontent.com/3875196/32850595-962cf84a-ca00-11e7-83b7-53861dc14822.png)

**Comment by varlesh** [comment]
if you mean blue color border - i disable this. Because it's not Adapta style and don't like me

**Comment by varlesh** [comment]
If you want blue border on window widget > use kvAdapta from kvantum  (it's maked by tsujan)

**Comment by kupiqu** [comment]
It's not only the blue line is also the background, which is darker

**Comment by varlesh** [comment]
Yes, used darker color on Breeze and? Why you compare Breeze engine with Kvantum? Kvantum it's another engine...

**Comment by tsujan** [comment]
@varlesh Some styles -- like Kvantum and QtCurve -- draw borders only when they're *explicitly* set because otherwise, there will so many ugly lines in a widget. To see the difference, compare Dolphin's view with that of Kate, or even Kate's side-pane with its view. From Kvantum's and QtCurve's perspective, Kate's side-pane has a border but its view doesn't. From Breeze and Fusion perspectives, both have similar borders (which isn't the case if you know that they're quite different things in Qt).

**Comment by kupiqu** [comment]
Sure, not complaining about that. I'm just mentioning that the transition is basically invisible, and that affects functionality. Just saying for the case there is a way to improve the functionality without breaking the theme philosophy...

**Comment by tsujan** [comment]
Let me be more specific here.

(1) This is from Qt source → `qtabwidget.cpp`:
```cpp
    if (q->documentMode())
        option->lineWidth = 0;
```
It says, if the tab-widget is in the document mode, the line-width is *zero*. Kvantum and QtCurve respect that but many styles don't. Kate's tab-widget is in the document mode too. If it wasn't, it would be like Dolphin's view and would have a border in Kvantum.

(2) Apart from that, Adpata's window color is near its base (view) color. You might see different things with other Kvantum's themes:

![kate2](https://user-images.githubusercontent.com/981076/32858074-68b6d660-ca5f-11e7-8bfb-7b0ada94d8bb.png)

(3) Kate's tab are styled in its code, which means that they don't follow the current style. This is a bad design, IMO, but other developers might have different opinions.

**Comment by kupiqu** [comment]
Thanks, I think this is really the ultimate issue, and it's not Adapta/Kvantum specific...
